user-guide/modules/ROOT/pages/explore-the-content.adoc

```diff
@@ -0,0 +1,120 @@
+
+
+== High-Profile users of Boost
+
+Some well known libraries and applications have used the building blocks that Boost provides. These include:
+
+* https://engineering.fb.com/2012/06/02/developer-tools/folly-the-facebook-open-source-library/[*Folly*], the Facebook Open Source Library, is an open-source library developed by Facebook that is intended to augment the Standard Library and Boost by providing additional features and optimizations, often with a focus on high-performance systems. It is a collection of relatively small and standalone libraries, and its features can be used on an as-needed basis.
++
+Some of these features include string formatting libraries, hash maps, dynamic types, and concurrency utilities. Folly makes extensive use of:
+
+    ** boost:context[] for creating and manipulating execution contexts, which are used in Folly's implementation of coroutines
+
+    ** boost:intrusive[] which provides containers that allow for the insertion and removal of elements in constant time
+
+    ** boost:optional[]  which provides a way to represent optional, or nullable, objects of a given type
++
+Additionally, many parts of Folly are designed to interact seamlessly with Boost and the Standard Library, making it easy to integrate Folly into existing codebases that use these libraries. Folly serves as an example of how Boost libraries can be used in the development of large-scale, high-performance systems. 
+
+* The https://bitcoin.org/en/bitcoin-core/[*Bitcoin Core*] client, which is the reference implementation of the Bitcoin protocol, makes extensive use of the Boost libraries. For example, it uses boost:asio[] for network programming, boost:thread[] for multithreading, and boost:test[] for unit tests.
+
+* https://www.mongodb.com/[*MongoDB*], a popular open-source document database, leverages libraries such as boost:filesystem[] for filesystem interactions, boost:thread[] for threading, and boost:smart_ptr[] for smart pointers.
+
+* The https://llvm.org/[*Low Level Virtual Machine*] (LLVM) project, which is a collection of modular and reusable compiler and toolchain technologies, utilizes Boost in parts of its code base. For instance, it uses boost:mpl[] for its metaprogramming functionality.
+
+NOTE: boost:mpl[] has now been superseded by boost:mp11[].
+
+* Microsoft uses Boost in several projects, including in https://visualstudio.microsoft.com/[*Visual Studio*], which uses boost:test[] for some of its unit testing.
+
+* Adobe also uses Boost in several of their software products. For instance, https://stlab.adobe.com/[*Adobe Source Libraries*] (ASL) includes and builds upon boost:smart_ptr[], boost.mpl[], boost:lambda[], and other libraries too.
```

Comment:
"and other libraries too" seems redundant since the sentence already starts with "For instance"

Reply:
Agreed, removed the phrase.